homeassistant/components/teslemetry/__init__.py

```diff
@@ -43,6 +46,14 @@
 
 type TeslemetryConfigEntry = ConfigEntry[TeslemetryData]
 
+CONFIG_SCHEMA = cv.empty_config_schema(DOMAIN)
+
+
+async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+    """Set up the Telemetry integration."""
+    # Register services here
+    return True
+
```

Comment:
This one isn't functional / used in this PR

```suggestion
```

Reply:
Removed.